### DISCUSSION TITLE
Where do you configured the alarm snapshots sensor?

### DISCUSSION THREAD

**Discussion by Headkillah** [jazzyisj/home-assistant-config#85] [General]
Hello @jazzyisj :)
First, Keep making such a good work 💯 

Now to my question:
Because i have many cams and save snapshots / videos, i wanted to use your part of your code for notification / purging the videos. BUT, where do you have configured the "sensor.alarm_snapshots" ?? I don´t find it nowhere in your code 😞

As example, you use it in the automation "[Hass] Purge Alarm Snapshots Folder" as an condition.

Thx in advance

**Comment by jazzyisj** [comment]
[Right here](https://github.com/jazzyisj/home-assistant-config/blob/00362266c83ff01a0d69258859fc8f22542e1ad6/packages/hass.yaml#L200).

```
  - platform: folder # https://www.home-assistant.io/integrations/folder/
    folder: /config/www/alarm_snapshots # path must be in allowlist_external_dirs
    filter: "*"
```

It's a [folder senor](https://www.home-assistant.io/integrations/folder/).

Glad you've found some useful stuff in there :-)

**Reply by Headkillah** [reply]
Oh man, and here I see again that I still have a lot to learn. I already have the TTS sensor active, I should have guessed :D

Many, many thx for your answer my friend :)

But, i have another question: I´m using your "Last person that comes / last person that goes" code. Mostly everything runs fine, but sometimes i have the problem, that the house mode does not change (e.g. from "away" to "home"). Do you have an idea?

**Reply by jazzyisj** [reply]
Use the automation / script trace viewer to see what happens when it doesn't work as expected.  

 I have also had an issue a couple of times since the last update, haven't had a chance to flush out the problem yet.  I think HA is running things faster now so maybe some timing is off somewhere now.  It will probably be a few days before I can dive into it.

**Comment by jazzyisj** [comment]
@Headkillah so my issue was with the Alarmo integration and the timing of when the state master vs various alarm zones changed.  Something changed in the last update so I had to make an adjustment to force my automation to use the master zone when appropriate.  

The other issue I'm having is my phone GPS location is taking a bit to update when I arrive home right now. I don't think that is related to HA at all, but rather my phone.

So probably not the issues you are having.   Did you figure your issue out?

**Reply by Headkillah** [reply]
@jazzyisj Not atm because of rare spare time :( But i´ve installed a fresh HA test installation and do cleanup most of the code, it´s the better way because of too much testing/ changing :)

I hope everything will getting better than :)
